source/dub/generators/build.d

```diff
@@ -645,3 +650,40 @@ unittest { // issue #1235 - pass no library files to compiler command line when
 	auto gen = new BuildGenerator(prj);
 	gen.generate(settings);
 }
+
+version (Posix)
+private string signalToName (int signal) {
+	import core.stdc.signal;
+	import core.sys.posix.signal;
+
+	enum sig_types = [
+		"SIGHUP",
+		"SIGINT",
+		"SIGQUIT",
+		"SIGILL",
+		"SIGABRT",
+		"SIGFPE",
+		"SIGKILL",
+		"SIGSEGV",
+		"SIGPIPE",
+		"SIGALRM",
+		"SIGTERM",
+		"SIGUSR1",
+		"SIGUSR2",
+		"SIGCHLD",
+		"SIGCONT",
+		"SIGSTOP",
+		"SIGTSTP",
+		"SIGTTIN",
+		"SIGTTOU",
+	];
+
+	final switch (signal)
```

Comment:
Hmm.. maybe this should be a switch with a default, I might have missed some signals here. Unfortunately I couldn't find a simple array/tuple of all signals anywhere in Druntime, they are all scattered around.